tools/test_merge_bot/config.js

```diff
@@ -0,0 +1,5 @@
+// A URL that, on GET, will return an array of objects with the schema
+// { round_id, datetime, test_merges, server, url }
+// You can see the moth.fans implementation in Rust here: https://github.com/Mothblocks/mothbus/blob/41fec056824edba0ffdfa39882b67739bf475d83/src/routes/recent_test_merges.rs#L30
+export const GET_TEST_MERGES_URL =
```

Comment:
Shouldn't this just be a value/secret passed down the script from the action ? Makes more sense to me that way instead of hard baking it in the source.

Reply:
Having it as a secret seems fine. I just wanted to make sure downstreams could change it. My thought process was they'd be free to ignore merge conflicts on a config.js.

Reply:
I'll make it a secret later today.